test/testdata/test_array_positions.csv

```diff
@@ -0,0 +1,10 @@
+0.000000000000000000e+00,0.000000000000000000e+00,0.000000000000000000e+00
```

Comment:
check with @nicrip on whether it is okay to publish this file?

Reply:
Oops sorry I thought the test files were in the repo already. Let's leave them out for now until I ask him.